src/main/app/src/app/shared/edit/edit-datum-groep/edit-datum-groep.component.html

```diff
@@ -54,23 +54,33 @@
             </div>
         </div>
     </div>
-    <div class="datum-control-wrapper">
-        <div class="time-control">
-            <button mat-icon-button color="primary" (click)="verlengen()" title="{{ 'actie.zaak.verlengen' | translate}}">
-                <mat-icon class="time-button">history</mat-icon>
-            </button>
-        </div>
-        <div *ngIf="!opgeschort" class="play-control">
-            <button mat-icon-button color="primary" (click)="opschorten()" title="{{ 'actie.zaak.opschorten' | translate}}">
-                <mat-icon class="play-icon">play_arrow</mat-icon>
-                <mat-icon class="play-button">pause_circle_outline</mat-icon>
-            </button>
+    <div class="datum-control-wrapper" fxLayout="row wrap" fxLayout.lt-sm="column wrap">
+        <div [fxFlex]="75">
+            <div *ngIf="verlengDuur" fxLayout.sm="row">
+                {{'duurVerlenging' | translate:{duur: verlengDuur} }}
```

Comment:
Is verlengDuur niet ook in dagen? Ik had verwacht dat deze regel nagenoeg hetzelfde zou zijn dan die bij opschorting